README.md

```diff
@@ -119,3 +119,55 @@ By default footnote anchors are expected to be surrounded by brackets. If it is
 ```
 --css '.fnanchor:before { content: "["; } .fnanchor:after { content: "]"; }'
 ```
+
+## Footnote extraction styles supported
+
+##### html: `class="footnote"`
+
+	<div class="footnote">
+	  <a id="Footnote_926" href="#FNanchor_926" class="label">[926]</a> Kidston (94), p. 250.
+	</div>
+
+##### Text output from formatting rounds (`"projectID"` prefix): `[Footnote #:` followed by 1 or more paragraphs, with end bracket, and possible `*[Footnote:` continuation blocks.
```

Comment:
These are really really long headers, can we make them shorter and move the content into normal text or something similar?